scripts/efficientnets/run.py

```diff
@@ -26,28 +26,38 @@
 def main(cfg: DictConfig = None):
     log.info("Training Configs:\n%s", OmegaConf.to_yaml(cfg))
 
+    width, _, img_size, dropout_p, _, _ = compound_params(cfg.name)
+    transforms = T.Compose(
+        [
+            T.Resize(size=(img_size, img_size)),
```

Comment:
you can leave like if h/w are the same size

`T.Resize(size=img_size),`

Reply:
I know, Just to make sure the image is resized in height also width